web/src/parts/account-item-cluster.comp.js

```diff
@@ -67,12 +69,12 @@ export function AccountItemCluster({address, id}) {
               </Button>
             </Td>
           ) : (
-            <Td isNumeric maxW="50px">
+            <Td isNumeric maxW="none">
```

Comment:
Is "none" the same as just omitting this attribute entirely?

Reply:
I think there is a default max width on these components that I need to override. Will double check.